transactions/WFLY-14869-microprofile-lra.adoc

```diff
@@ -0,0 +1,128 @@
+= Support for MicroProfile LRA
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:keywords:          transaction, saga, lra, microprofile
+
+== Overview
+
+The goal of this RFE is providing the Galleon feature pack for WildFly which integrates the
+https://github.com/jbosstm/narayana/tree/master/rts/lra[Narayana implementation] of the
+https://github.com/eclipse/microprofile-lra[MicroProfile LRA specification].
+
+The MicroProfile LRA is a transaction processing pattern well suited for microservice development based on the saga
+pattern. The motivation of such processing is summarized in the
+https://github.com/eclipse/microprofile-lra/blob/1.0/README.adoc#motivation[MicroProfile Spec readme].
+
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-14869
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1496
+
+=== Dev Contacts
+
+* mailto:mstefank@redhat.com[{author}]
+
+=== QE Contacts
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+
+* [x] Engineering
+* [ ] QE
+
+
+=== Affected Projects or Components
+
+* Transactions
+* WildFly
+
+=== Other Interested Projects
+
+* MicroProfile
+* Galleon
+* WildFly Operator
+
+=== Relevant Installation Types
+
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+* [ ] Managed domain
+* [ ] OpenShift s2i
+* [x] Bootable jar
+
+== Requirements
+
+=== Hard Requirements
+
+The feature pack will integrate the LRA specification as the Galleon feature pack.
```

Comment:
AFAICT the feature pack will actually integrate the Narayana implementation of the LRA spec into a new Galleon feature pack, right?

Reply:
Maybe I didn't make my point. What hit my eye is that the feature pack should be integrating the implementation, rather than the specs. Or at least both, IIUC. But I might be missing something. WDYT?

Reply:
I don't think there is any new feature pack here. If we're talking a new feature pack, this would stay in WildFly Extras.  This needs to be about adding capabilities to the existing 'wildfly' feature pack.